contents.json

```diff
@@ -946,7 +946,7 @@
       "title": "swift-package-manager",
       "category": "dependency-managers",
       "description": "SPM is the Package Manager for the Swift Programming Language.",
-      "homepage": "https://github.com/apple/swift-package-manager"
+      "homepage": "https://github.com/swiftlang/swift-package-manager"
```

Comment:
You can see the CI failure that led to these updated links here: https://github.com/matteocrippa/awesome-swift/pull/1846/checks?check_run_id=31793504943

Reply:
thanks